string client_id
int64 req_id
string param
bool value
---
string client_id
int64 req_id
